### DISCUSSION TITLE
添加阿里云盘open 出现 Post "https://open.aliyundrive.com/adrive/v1.0/user/getDriveInfo": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

### DISCUSSION THREAD

**Discussion by jerryc127** [AlistGo/alist#4138] [Q&A]
添加阿里云盘open 出现 Post "https://open.aliyundrive.com/adrive/v1.0/user/getDriveInfo": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
![image](https://user-images.githubusercontent.com/16351105/231702663-6e922ac2-08ce-49d6-85b0-4d2d4fbe6cd8.png)

**Comment by ken357525161** [comment]
我今天也遇到这个问题，至今也没修复？？汗
